.github/workflows/publish.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         with:
           path: | 
             **/node_modules
-          key: node-modules-${{ runner.os }}-${{env.NODE_VERSION}}-${{hashFiles('**/package.json', '**/package-lock.json')}}
+          key: node-modules-${{runner.os}}-${{steps.get_node_version.outputs.NODE_VERSION}}-${{hashFiles('**/package.json', '**/package-lock.json')}}
```

Comment:
For consistency consider pouring the node_version into an environment variable as in the other workflow. 

I know it is only used once in this workflow, but if it is to be extended, I think it is a nice way to nudge a future contributor, into using environment variables instead of `steps.get_node_version.outputs.NODE_VERSION`